examples/systemcalls/systemcalls.c

```diff
@@ -57,11 +62,42 @@ bool do_exec(int count, ...)
  *   (first argument to execv), and use the remaining arguments
  *   as second argument to the execv() command.
  *
-*/
+ */
+
+  if(command[0][0] != '/') 
+  {
+    return false;
+  }
+
+  int status;
+  pid_t pid = fork();
 
-    va_end(args);
+  if(pid == -1) 
+  {
+    return false;
+  } 
+  else if(pid == 0) 
+  {
+    int returnValue = execv(command[0], command);
 
-    return true;
+    if(returnValue == -1)
+    {
+      return false;
```

Comment:
@qrogers6 need to `exit` instead of `return` here, otherwise the child thread will continue to run the test case when execv fails.  You can remove your `command[0][0] != '/'` check if you exit here.